msys2/PKGBUILD

```diff
@@ -36,17 +35,21 @@ build() {
 
   mkdir build
   cd build
-  MSYS2_ARG_CONV_EXCL=-DCMAKE_INSTALL_PREFIX= cmake \
-    -G "MSYS Makefiles" \
+
+  CXXFLAGS+=" -Wno-format" \
+  MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
+  ${MINGW_PREFIX}/bin/cmake.exe \
+    -G "Ninja" \
     -DCMAKE_INSTALL_PREFIX="${MINGW_PREFIX}" \
     -DBUILD_TESTING=OFF \
     ../
-  cmake --build .
+
+  ${MINGW_PREFIX}/bin/cmake.exe --build ./
 }
 
 package() {
   cd "${srcdir}"/../../build
-  make DESTDIR="${pkgdir}" install
+  DESTDIR="${pkgdir}" ${MINGW_PREFIX}/bin/cmake.exe --build ./ --target install
```

Comment:
`DESTDIR="${pkgdir}" cmake --build ./ --target install`?

Reply:
Yes, previous command works. The new one is just to play safely.